R/program_config_module.R

```diff
@@ -0,0 +1,132 @@
+
+#' UI part for program config
+#'
+#' Creates two panels for the user to choose the paths to the DYNAMO executable
+#' and to the working directory.
+#'
+#' @param id The id for reference in the server.
+#'
+#' @returns Shows an error message when the reference data is not NULL and is
+#' does not have the right content.
+#'
+#' @export
+program_config_ui <- function(id) {
```

Comment:
Perhaps rename `id` to `namespace_id` and give a bit more explanation in the docstring?

Reply:
I left the argument as `id`; changing it did not affect the app but the tests and I did not want investigate how to fix it (see also here: https://stackoverflow.com/questions/64391273/how-to-test-modules-within-a-golem-framework-shiny-package). I think using the argument name `id` is consistent with conventions in shiny. But, I did update the docstring to make it more precise that it's the namespace id.